.github/workflows/deploy-prod.yaml

```diff
@@ -0,0 +1,17 @@
+name: Deploy to prod
+on: [workflow_dispatch]
+jobs:
+  deploy_prod:
+    if: github.repository_owner == 'ParkingReformNetwork'
+    runs-on: ubuntu-latest
+    steps:
+    - name: Copy from staging to production
+      uses: appleboy/ssh-action@master
+      with:
+        host: ${{ secrets.SERVER_HOST }}
+        username: ${{ secrets.SERVER_USERNAME }}
+        key: ${{ secrets.SERVER_PRIVATE_KEY }}
+        script: |
+          rsync --archive --verbose --compress --delete \
```

Comment:
`--delete` will remove from the prod folder anything that is not in the staging folder. That is, it removes leftover files.